src/Configuration/RedisConfiguration.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Equip\Configuration;
+
+use Auryn\Injector;
+use Redis;
+
+class RedisConfiguration implements ConfigurationInterface
+{
+    use EnvTrait;
+
+    /**
+     * @inheritDoc
+     */
+    public function apply(Injector $injector)
+    {
+        $injector->prepare(Redis::class, function($redis) {
+            $redis->connect(
```

Comment:
HHVM builds it in. If someone wanted to use Predis they could write their own configuration.